terra-app-local.sh

```diff
@@ -0,0 +1,285 @@
+#!/bin/bash
```

Comment:
This kinda turned into a big bash script, kinda thinking about other CLI tooling like python as alternative

Reply:
FWIW it doesn't look bad to me yet. Also, I'd wonder if invoking those tools may not be as convenient in another language. I don't feel strongly though.

Reply:
Yeah.. that's why I picked bash and not something like ammonite which I think would cause more pain than it's worth.